CLIENT-SPECIFICATION.md

```diff
@@ -212,6 +212,9 @@ Caching SHOULD be done according to the user's language configuration (if any),
 
 ## Changelog
 
+ - [v1.5, March 11th 2021](https://github.com/tldr-pages/tldr/blob/8505e55b568dbc394ea853d488cd07484c051e21/CLIENT-SPECIFICATION.md) ([#5327](https://github.com/tldr-pages/tldr/pull/5327))
+   - Add requirement for converting command names to lowercase before searching for them.
```

Comment:
Perhaps this will read better?
```suggestion
   - Add requirement for converting command names to lowercase before running the page resolution algorithm.
```